ci/bump_version.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash -xv
+
+if [ $# != 1 ]
+then
+     echo "Please provide version as the first argument"
+     exit 1
+fi
+
+version=$1
+CI_DIR="$( cd "$( dirname "$0" )" && pwd )"
+file=Cargo.toml
+final_exit_code=0
+
+declare -a crates=(
+  "automap"
+  "dns_utility"
+  "masq"
+  "masq_lib"
+  "multinode_integration_tests"
+  "node"
+  "port_exposer"
+)
+declare -a failed_crates
+
+bump_version() {
+  pushd "$CI_DIR/../$1"
+
```

Comment:
I notice that if `sed` does not find a `version` line to replace, the script will not know, and the user will be unaware that a `Cargo.toml` file remains unchanged.

You could first try `grep`ping for the same expression you're trying to change with `sed` (pipe the result to `/dev/null`, because you don't care about anything but whether there was output or there wasn't), and looking at the exit code. If `grep` found something, it'll be 0; if not, it'll be 1. If you get a 1 from `grep`, you can put the file straight into the failed list and not bother `sed`ding it.